pandas/tests/indexes/interval/test_interval_tree.py

```diff
@@ -154,14 +154,14 @@ def test_get_indexer_closed(self, closed, leaf_size):
             (np.array([0, 2, np.nan]), np.array([1, 3, np.nan]), False),
         ],
     )
-    @pytest.mark.parametrize("order", map(list, permutations(range(3))))
+    @pytest.mark.parametrize("order", map(lambda x: list(x), permutations(range(3))))
```

Comment:
as this is a known issue with mypy, see https://github.com/python/mypy/issues/6697, i'd prefer not to change the code here.

add a `# type: ignore` and add the issue number as a comment i.e.

```python
    # https://github.com/python/mypy/issues/6697
    # error: Argument 1 to "map" has incompatible type "Type[List[Any]]";
    #  expected "Callable[[Tuple[int, ...]], List[_T]]"
    @pytest.mark.parametrize("order", map(list, permutations(range(3))))  # type: ignore
```